runtime/modules/java/core/src/main/java/io/atlasmap/java/inspect/ClassInspectionService.java

```diff
@@ -777,15 +699,15 @@ protected Integer detectArrayDimensions(Class<?> clazz) {
         return modifiers;
     }
 
-    protected Class<?> detectListClass(ClassLoader classLoader, Field field) throws ClassNotFoundException {
+    private Class<?> detectListClass(ClassLoader classLoader, Field field) throws ClassNotFoundException {
         List<String> types = detectParameterizedTypes(field, true);
         if (types != null && !types.isEmpty()) {
             return classLoader.loadClass(types.get(0));
         }
         return null;
     }
 
-    protected Class<?> detectArrayClass(Class<?> clazz) {
+    Class<?> detectArrayClass(Class<?> clazz) {
```

Comment:
why package private?

Reply:
out this class it is referenced only in the testcases. Shall I use the protected in these kind of cases